stingray/lightcurve.py

```diff
@@ -1628,10 +1479,10 @@ def from_lightkurve(lk, skip_checks=True):
     def to_astropy_timeseries(self):
         return self._to_astropy_object(kind="timeseries")
 
-    def to_astropy_table(self):
-        return self._to_astropy_object(kind="table")
+    def to_astropy_table(self, **kwargs):
```

Comment:
Needs docstring? Especially since it now takes `**kwargs` and the user might want to know what keyword arguments they can include. :)